docs/en/config_mc/mc_jerk_limited_type_trajectory.md

```diff
@@ -4,12 +4,6 @@ The Jerk-limited trajectory type provides smooth motion in response to user stic
 It generates symmetric smooth S-curves where the jerk and acceleration limits are always guaranteed.
```

Comment:
The line above says " provides smooth motion in response to _user stick input_" - but isn't this only used in missions now?